src/reducers/fiatToCryptoReducer.js

```diff
@@ -18,34 +18,89 @@
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
 
-import { SET_ALTALIX_INFO } from 'constants/fiatToCryptoConstants';
+import {
+  SET_ALTALIX_INFO,
+  SET_SENDWYRE_RATES,
+  LOAD_SENDWYRE_COUNTRY_SUPPORT,
+  SET_SENDWYRE_COUNTRY_SUPPORT,
+  RESET_SENDWYRE_COUNTRY_SUPPORT,
+  SENDWYRE_SUPPORT,
+} from 'constants/fiatToCryptoConstants';
 
-type AltalixInfo = {
-  isAvailable: boolean,
-};
+import type { SendwyreRates } from 'models/FiatToCryptoProviders';
 
-export type FiatToCryptoReducerState = {
-  altalix: null | AltalixInfo,
-};
+export type FiatToCryptoReducerState = {|
+  isAltalixAvailable: null | boolean,
+  sendwyreExchangeRates: null | SendwyreRates,
+  sendwyreCountrySupport: $Values<typeof SENDWYRE_SUPPORT>,
+|};
 
 type AltalixInfoSetAction = {
   type: typeof SET_ALTALIX_INFO,
-  payload: AltalixInfo,
+  payload: { isAvailable: boolean },
```

Comment:
I wonder if we really need `isAvailable` or we can just leave `boolean` value for payload?

Reply:
It got left over from where I flattened state to avoid making the reducer more complex. It's fixed now.